### ISSUE TITLE
Merge script does not deal with multiple fixes

### ISSUE THREAD

**Issue by philwebb** [spring-projects/spring-boot#17229]
A merge commit with multiple Fixes/Closes message only seems to get a forward-port for the first item.

**Comment by mhalbritter** [comment]
I just tested it again. I've commited on 2.7.x a commit with this message:

```
Closes gh-1
Closes gh-2
```

Then i ran on `main` a `git merge 2.7.x`:

```
╰─$ git merge 2.7.x             
Dry run
POSTing to https://api.github.com/repos/spring-projects/spring-boot/issues with body {"title":"Fixed name of annotation and a typo in the code.","labels":["status: forward-port"],"milestone":281,"body":"Forward port of issue #1 to 3.0.x."}
Created gh-dry-run for forward port of gh-1 into 3.0.x
Dry run
POSTing to https://api.github.com/repos/spring-projects/spring-boot/issues with body {"title":"[BS-157] Add support for MultipartConfig","labels":["status: forward-port"],"milestone":281,"body":"Forward port of issue #2 to 3.0.x."}
Created gh-dry-run for forward port of gh-2 into 3.0.x
Merge made by the 'ort' strategy.
```

So it looks like that this works, or did i miss something?

**Comment by philwebb** [comment]
Looks like it was fixed in #25586
